src/main/java/org/opentripplanner/middleware/persistence/TypedPersistence.java

```diff
@@ -122,6 +122,16 @@ public T getById (String id) {
         return mongoCollection.find(eq(id)).first();
     }
 
+    /**
+     * Retrieves the first document that matches an field value, or null if there is no match.
+     * @param field The field to scan.
+     * @param value The value to find.
+     * @return A document whose specified field has the specified value.
+     */
+    public T getByField (String field, String value) {
```

Comment:
I would like this to be named `getFirstByFieldValue` to be clear that it is returning a single record and getting the first result whatever that may be.

Reply:
I think this method will become `getOneFiltered` once #14 is merged.

Reply:
Method is redundant. I went ahead and removed it.